virTemplate.md

```diff
@@ -0,0 +1,65 @@
+
+The virus named dicistroviridae was found in the sample runs SRR13764152 (GSM5100145: P90p5_Em) and SRR13764153 (GSM5100146: P90p6_Eff). This was determined by copying the index run into serratus.io [@edgar2022petabase]. Both of these samples had many more reads (around 100M) while the other samples of the same CD8+ TIL cell type had less reads (around 20M) thus it may have only been detected in these Eff and Em samples since they had higher read counts. This virus was not detected in the Ebo subset of cells (it was not in SRR13764151 nor the other Ebo datasets). This dicistroviridae is highly related to the ancient Northwest Territories cripavirus. Cripavirus is within the dicistroviridae family. After doing a blastx search on the transcript from SRR13764152 which contains the palmprint of the dicistrovirus, the top results were Rdrp for picornavirales and dicistroviridae is in the picornavirales order. A dicistrovirus Rdrp did not arise as the top hit which confirms that this is a novel virus that has not yet been characterized. 
+
+For the SRR13764151 (Ebo) and SRR13764152 (Em) dataset, the analysis tab of the metadata shows the taxonomical species in the sample. However, unfortunately for the SRR13764153 (Eff) dataset, it does not allow me to see the taxonomy. For both SRR13764151 and SRR13764152 datasets, I see Plasmodium was detected (the RNA sequence of Plasmodium was detected). This parasite is transferred from mosquitoes to humans [@churcher2017probability]. In addition, discistrovirdiae are known to only infect insects [@steward2013marine]. In a study by Fauver et al., dicistrovirus was found in the Anopheles species of mosquitoes [@fauver2016west]. Female Anopheles mosquitoes are known to be hosts of Plasmodium [@mota2019parasitism]. Therefore, it is possible that the humans were bit by Anopheles mosquitoes who then transferred a dicistrovirus into them. Although Plasmodium is also in SRR13764153 (Ebo) but dicistrovirus is not in those cells, the Ebo cells may have a phenotype that prevents the dicistrovirus from being able to infect or replicate in that cell.
+
+### Virus Genome {Q3}
```

Comment:
Remove Q3 from sub-title, and worth changing "Virus Genome" to "Dicistrovirus Genome Fragment" which is better at communicating this is not a complete genome.